prod-sample/sample-modules/frank-lloyd-root/0.0.3/src/components/FrankLloydRoot.jsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable global-require */
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import { fromJS } from 'immutable';
+import ModuleRoute from 'holocron-module-route';
+import { holocronModule } from 'holocron';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import HelloWorldComponent from './HelloWorld';
+import Partial, { onPartialRouteEnter } from './Partial';
+
+export const defaultIcon = 'http://localhost:3001/static/modules/frank-lloyd-root/0.0.3/assets/pwa-icon-192px.png';
```

Comment:
what about ? localhost might cause some issues with the remote integration tests
```suggestion
export const defaultIcon = `${config.get('cdnUrl')}modules/frank-lloyd-root/0.0.3/assets/pwa-icon-192px.png`;
```

Reply:
@JAdshead do our integration tests use a dynamic port?

Reply:
I've updated this manually as it would need to be in the component scope for config (97d3156).

Reply:
@JamesSingleton I believe the `browserUrl` stays consistent but the `fetchUrl` (used on `localhost`) does have a dynamic port assigned during integration tests.